ext/drbbdup/drbbdup.c

```diff
@@ -665,8 +665,7 @@ drbbdup_is_at_end(instr_t *check_instr)
 }
 
 /* Returns true if at the start of the end of a bb version: if check_instr is
- * the start emulation label for the inserted jump or the exit label.  If there
- * are no emulation labels this is equivalent to drbbdup_is_at_end().
+ * the start emulation label for the inserted jump or the exit label.
```

Comment:
How about adding that this does not return true for certain types of blocks like not ending in a branch/syscall or having unending emulation like repstr.

Reply:
Added a comment.